.gitignore

```diff
@@ -32,4 +32,7 @@ CMakeLists.txt.user
 
 # Other Stuff
 tags
-*.d
\ No newline at end of file
+*.d
+
+# Generated CMake file
+cmake/modules/CPackREADME.cmake
```

Comment:
Can you generate this file in the build directory? The source dir should not be touched.